tools/doc/html.js

```diff
@@ -307,14 +329,19 @@ function buildToc(lexed, filename, cb) {
     depth = tok.depth;
     const realFilename = path.basename(realFilenames[0], '.md');
     const id = getId(realFilename + '_' + tok.text.trim());
+
     toc.push(new Array((depth - 1) * 2 + 1).join(' ') +
-             '* <a href="#' + id + '">' +
-             tok.text + '</a>');
+             '* %stability_' + tok.stability + '%' +
```

Comment:
@Fishrock123 sorry, but this code cannot be implemented, because `<li>` html tag generated by [this](https://github.com/nodejs/node/pull/7189/files#diff-de394d26dd6b5ebee9f95daaeaf7e5e1R340) code.

Line
```
+             '* %stability_' + tok.stability + '%' +
```
need because need to added `class`

Also, `<li>` don't need to be closed, because tag generates by markdown parser.